wallet/wallet-rpc-lib/src/rpc/mod.rs

```diff
@@ -225,6 +231,115 @@ impl<N: NodeInterface + Clone + Send + Sync + 'static> WalletRpc<N> {
         Ok(NewAccountInfo::new(num, name))
     }
 
+    pub async fn add_standalone_watch_only_address(
```

Comment:
Shouldn't we resync when adding these new addresses?

Reply:
added

Reply:
Same for multisig is not done, I believe.

Reply:
Please print a message after syncing starts... there's no way to recognize that. Something like "Rescanning the blockchain to detect balance in added new addresses"